utils.py

```diff
@@ -29,3 +31,23 @@ def bbox_to_coordinates(bbox: np.ndarray, rot: bool = True):
 
     vertices = np.asarray(vertices)
     return vertices
+
+
+def calc_iou(box1: List[Tuple[int, int]], box2: List[Tuple[int, int]]) -> float:
+    """
+    Calculate IoU of two boxes presented by their corners' coordinates
+    :param box1: list of 4 tuples of (x, y) coordinates: [left_top, right_top, right_bottom, left_bottom]
+    :param box2: list of 4 tuples of (x, y) coordinates: [left_top, right_top, right_bottom, left_bottom]
+    :return: intersection over union score
+    """
+
+    x1 = max(box1[0][1], box2[0][1])  # left top angle of the intersection
+    y1 = max(box1[0][0], box2[0][0])  #
+    x2 = min(box1[2][1], box2[2][1])  # right bottom angle of the intersection
+    y2 = min(box1[2][0], box2[2][0])  #
```

Comment:
Rename in the manner of `bbox_to_coordinates()`, then comments will be redundant.